examples/QIR/Development/Development.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.LlvmBindings.Native" Version="13.0.0-CI-20220129-005156" PrivateAssets="All" GeneratePathProperty="true"/>
+    <PackageReference Include="Microsoft.Quantum.LlvmBindings.Native" Version="13.0.0-CI-20221216-194411" PrivateAssets="All" GeneratePathProperty="true"/>
```

Comment:
Oops, this was a chicken and egg issue in my PR to main that I thought I already resolved. I can fix that in main rather than try to address it here, since this is just merging the contents of main into the feature branch.